forms-flow-web/src/customFormioComponents/FileUpload/fileUploadInputComponent.jsx

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+
+import React, { Component } from 'react';
+
+export default class UploadFile extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedFile: '',
+            name: ""
+        };
+    }
+    onFileChange = (event) => {
+        let file = event.target.files[0];
+        console.log(file);
+        this.setState({ selectedFile: file });
+    };
+
+
+    onFileUpload = () => {
+
+        var timestamp = new Date().getTime();
+        var fileName = timestamp + "-" + this.state.selectedFile.name;
```

Comment:
as we are using ES6 syntax use let / const instead of var unless there are specific reasons for it.

Reply:
Moving it later